4-HTTP/server/index.js

```diff
@@ -0,0 +1,38 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const port = 3000;
+
+app.use(cors());
+app.use(
+  express.urlencoded({
+    extended: true,
+  })
+);
+app.use(express.json());
+
+let list = [];
+
+app.get("/", (req, res) => {
```

Comment:
I would prefix it with some meaningful name:
```suggestion
app.get("/todos", (req, res) => {
```

Reply:
OK